README.md

```diff
@@ -63,9 +63,26 @@ Here's a step-by-step description of how a game of Tateddrez would unfold:
 * Any external module/plugin/library/resource should be in the project. Please don't use github URLs to intstall UPM packages.  
 ---
 # Delivery
-* Fork this repository or clone this repository and create a new one into your github account and share the repository with the users "shanickgauthier", "juanblasco" and "kk-homa" or make it public.
+* Fork this repository or clone this repository and create a new one with your github account. Share the new repository with the users "shanickgauthier", "juanblasco" and "kk-homa" or make it public.
 * Build Android .apk and upload it to the repository.  
 ---
+# What we will be evaluated?  
+* Additional documentation    
+* Project folder structure  
+* General code architecture  
+* Scalability  
+* Single responsibility principle
```

Comment:
We may say SOLID principles instead of listing only SRP and OCP

Reply:
I'd prefer to leave the evaluation system as is to keep the previous scores intact. 
Also it could be challenging to find application of L or I principles within this compact case study.